README.rst

```diff
@@ -45,12 +45,16 @@ Currently, these hooks are available:
 Add required directories
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you are using any custom templates or licenses, copy the .reuse and LICENSES directories from this repository
-into the target repository. If you are using the default Ansys template and MIT.txt license, skip this step. By default,
-the hook will copy the LICENSES/MIT.txt, .reuse/templates/ansys.jinja2, and .reuse/dep5 into the target repository when
-the hook runs.
+If you are using the ansys.jinja2 template and MIT.txt license, skip this step. By default, the hook will make symbolic links
+from its "assets" directory containing LICENSES/MIT.txt and .reuse/templates/ansys.jinja2
+to your repository when the hook runs. The .reuse and LICENSES directories will be deleted once the hook is
+done running.
 
-The LICENSES and .reuse folders are required for the hook to run correctly. Your project should have the following layout:
+If you are using a custom template, create a directory named ".reuse", and if you are using a custom license, create a directory
+named "LICENSES" in the root of your repository. The custom template cannot be named ansys.jinja2, otherwise it will be removed
```

Comment:
```suggestion
named ``LICENSES`` in the root of your repository. The custom template cannot be named ``ansys.jinja2``, otherwise it will be removed
```